src/debugHighlight.ts

```diff
@@ -23,9 +23,8 @@ import * as vscodeTypes from './vscodeTypes';
 
 export type DebuggerError = { error: string, location: Location };
 
-const debugSessions = new Map<string, vscodeTypes.DebugSession>();
-
 export class DebugHighlight {
+  private _debugSessions = new Map<string, vscodeTypes.DebugSession>();
```

Comment:
note: reason for moving it out was that the tests were not isolated before, the `_debugSession` struct leaked.